staging/kos/cmd/attachment-tput-driver/README.md

```diff
@@ -72,25 +72,39 @@ to stdout and those files.
 
 ## Timing and threading
 
-This driver aims to issue requests to create and delete
-NetworkAttachment objects steadily, at a given rate.  Once a
-NetworkAttachment becomes "ready" it is normally tested with ping, but
-this can optionally be disabled.
+The driver defines flags that let users customize the timing with which
+requests to create and delete NetworkAttachments are issued.  Currently
+operations on NetworkAttachments can either happen with a constant period
+or according to a Poisson process; in both cases a driver parameter lets
+users specify the rate of operations.  Once a NetworkAttachment becomes
```

Comment:
Since we're now allowing a non-deterministic arrival process, what the user specifies is the _average_ rate of operations.